CONTRIBUTING.md

```diff
@@ -0,0 +1,9 @@
+# Contributing to JupyterHub
+
+Welcome! As a [Jupyter](https://jupyter.org) project,
+you can follow the [Jupyter contributor guide](https://jupyter.readthedocs.io/en/latest/contributing/content-contributor.html).
+
+Make sure to also follow [Project Jupyter's Code of Conduct](https://github.com/jupyter/governance/blob/HEAD/conduct/code_of_conduct.md)
+for a friendly and welcoming collaborative environment.
+
+If you need some help, feel free to ask on [Gitter](https://gitter.im/jupyterhub/jupyterhub) or [Discourse](https://discourse.jupyter.org/).
```

Comment:
Let's also open an issue about creating proper contributing instructions once the project is a bit more mature.

Reply:
Done